src/_community/openings.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Openings
+description: Current Openings for positions on the Quilt Staff Team
+
+redirect_from:
+  - /community/openings.html
+---
+The Quilt Project is very community-oriented, and that means that it needs your help to succeed. This page is a list of all the places we currently need help with, both in development and in our community.
+
+{% admonition %}
+**Please note:** The listed positions typically do not have a quota. As long as a position remains listed in this channel, we'll continue to be interested in reviewing and accepting more members for them.
+{% endadmonition %}
+
+## Development Roles
+
+Development teams build the tools that make up Quilt, from build tools, to mappings, to libraries, to loaders. Joining a development team is a great way to contribute if you have a knowledge of Java, and how Minecraft works behind the scenes.
+
+### Build Tools
+
+The Build Tools team is responsible for building the tools that make Quilt modding possible. This is one of our most important teams, and it maintains projects that are instrumental for getting Quilt to a full release.
+
+This team is currently focused on improving Sponge's VanillaGradle and bringing it into a state where it can be used for Quilt. Until this is completed, Quilt projects are reliant on Fabric's Loom and a set of additional Gradle plugins, so we're hoping to prioritise development in this team.
+
+If you're interested in helping out with Quilt's build tools or would like to join this team, please send a message in #build-tools-general on the [Toolchain Discord server](https://discord.quiltmc.org/toolchain).
```

Comment:
Did we ever figure out whether build tools wanted to be on the list?

Reply:
Yes, I asked Glitch and he said that it changes frequently but at the moment he did want to be on the list.